modules/developing/pages/using-curl.adoc

```diff
@@ -1,98 +1,70 @@
-|===
-
-Locate the value for the "Web Service URL" and copy the address to a safe place.
-
-== Retrieving your Pulsar token in Astra Streaming
+Web Service URLs start with `http`.
+Broker Service URLs start with `pulsar(+ssl)`.
+====
 
-Next, you need an authentication token.
-Most APIs use *tokens* to authenticate a request, and Pulsar follows this same pattern.
+== Create a {product} Pulsar token
 
-[NOTE]
+[IMPORTANT]
 ====
-Don't confuse your Pulsar token with your Astra token - the concept is the same, but it's a different authentication. xref:operations:astream-token-gen.adoc[More details are here].
+An xref:operations:astream-token-gen.adoc[{astra_db} application token] is _not_ the same as a Pulsar token.
 ====
 
-To create a new Pulsar token in your Astra Streaming tenant, navigate to the "Settings" tab in the Astra portal.
-|===
-a|image:settings-tab.png[Settings tab in Astra Streaming]
-|===
+For information about creating Pulsar tokens, see xref:operations:astream-token-gen.adoc[].
 
-Click "Create Token" and choose the time to expire.
-Be smart about this choice - "Never Expire" isn't always the best option.
-A window will appear with the newly generated token - it's a long string of letters and numbers.
+== Make a request
 
-|===
-a|image:copy-token.png[Copy token in Astra Streaming]
-|===
-
-Click the clipboard icon to copy the token to your clipboard, and paste the token in a safe place.
-This is the only time you will be able to copy it.
-
-== Example curl call
-
-You now have the two necessary ingredients to make RESTful calls to your Astra Streaming tenant. Let's put them to use!
-
-Set the required variables in a terminal window.
-
-[source,shell,subs="attributes+"]
+. Set your environment variables:
++
+[source,shell,subs="+quotes"]
 ----
-PULSAR_TOKEN="<REPLACE-ME-WITH-TOKEN>"
-WEB_SERVICE_URL="<REPLACE-ME-WITH-URL>"
+PULSAR_TOKEN="**PULSAR_TOKEN**"
+WEB_SERVICE_URL="**TENANT_WEB_SERVICE_URL**"
 ----
 
-Run the following curl command to list all the built-in sink connectors.
-
-[source,shell,subs="attributes+"]
+. Run a curl command.
+The following example lists built-in sink connectors.
++
+[source,curl]
 ----
-curl -sS --fail --location --request GET \
-  -H "Authorization: $PULSAR_TOKEN" \
-  "$WEB_SERVICE_URL/admin/v3/sinks/builtinsinks"
+curl -sS --location -X GET "$WEB_SERVICE_URL/admin/v3/sinks/builtinsinks" \
+--header "Authorization: $PULSAR_TOKEN" \
+--header "Content-Type: application/json"
 ----
 
-The output should be a very long JSON-formatted string containing details about all the sink connectors waiting for you to have fun with {emoji-smile}.
+== Format responses
 
-TIP: If your curl command doesn't go so smoothly, add `-v` to your call. +
-The `-v` is for verbose, and prints more detail about what problem has occurred.
+The default response is a single JSON string.
 
-== What's next?
+You can use modifications like ` | jq .` or ` | python3 -mjson.tool` to format the output for easier reading.
```

Comment:
Is the spacing correct for the code ticks?

https://d5rxiv0do0q3v.cloudfront.net/doc-4449-streaming-audit/astra-streaming/developing/using-curl.html#format-responses